src/components/Login/LoginForm.js

```diff
@@ -43,17 +43,37 @@ export default class LoginModal extends React.Component {
           console.log(err);
           let newMsg = err.message;
           switch (err.code) {
+            case "auth/invalid-email":
+              newMsg =
+                "Invalid username inputted. Usernames must only have alphanumeric characters plus !@#$%.";
```

Comment:
Technically shouldn't happen bc this is for creating a user but just in case, remove the inputted like above.